test/common/run-tests

```diff
@@ -430,6 +436,17 @@ def run(opts, image):
         if not made_progress:
             time.sleep(0.5)
 
+    # Create coverage report
+    if opts.coverage:
+        base_dir = os.path.realpath(f'{__file__}/../../..')
+        output = os.environ.get("TEST_ATTACHMENTS", base_dir)
+        lcov_files = glob.glob(f"{base_dir}/lcov/*.info.gz")
+        if len(lcov_files) > 0:
+            try:
+                subprocess.check_call(["genhtml", "--no-function-coverage", "--output-dir", f"{output}/Coverage"] + lcov_files)
+            except Exception as e:
```

Comment:
`subprocess.CalledProcessError` -- we want to crash on other errors.

Reply:
Done.